src/internal/observable/bindCallback.ts

```diff
@@ -144,7 +144,7 @@ export function bindCallback(callbackFunc: Function, scheduler?: SchedulerLike):
  * }
  *
  * const boundSyncFn = bindCallback(iCallMyCallbackSynchronously);
- * const boundAsyncFn = bindCallback(iCallMyCallbackSynchronously, null, Rx.Scheduler.async);
+ * const boundAsyncFn = bindCallback(iCallMyCallbackSynchronously, null, Rx.asyncScheduler);
```

Comment:
The `Rx` should be removed and `asyncScheduler` should to be added to the import:

```ts
import { bindCallback, asyncScheduler } from 'rxjs';
```

Reply:
I did it myself. LGTM otherwise.